geopyspark/geotrellis/catalog.py

```diff
@@ -453,3 +458,94 @@ def write(uri,
                                     tiled_raster_layer.srdd,
                                     TimeUnit(time_unit).value,
                                     IndexingMethod(index_strategy).value)
+
+
+class Attributes(object):
```

Comment:
This could be private. Do you think its cleaner to make it `_Attributes` or an inner class to `AttributeStore` ? Its basically impossible to have an `Attributes` without an instance of `AttributeStore` so inner class may be appropriate?